docs/source/deploying/configuration.rst

```diff
@@ -122,6 +122,24 @@ Quetz can store package in object cloud storage compatible with S3 interface. To
 :bucket_prefix:
 :bucket_suffix: channel directories on S3 are created with the following semantics: ``{bucket_prefix}{channel_name}{bucket_suffix}``
 
+``redis`` section
+^^^^^^^^^^^^^^^^^^^^^^
+
+Quetz can be configured to run with a redis-server and use redis-queue for queueing jobs and processing them in the background with workers.
+Note that you might need to run ``redis-server`` and ``rq worker`` commands in separate terminals / containers along with quetz.
```

Comment:
can you create a new file in docs/source/deploying/workers.rst describing the procedure step by step, including the installation and configuration of redis (you can also pull redis from docker).

the config options should be also given  here.

Reply:
you can also mention the other worker types. If you don't have time, I can also do this after this PR is merged.

Reply:
We should do this in a separate PR.

Reply:
Done via https://github.com/mamba-org/quetz/pull/202